components/motor/gpio/encoded.go

```diff
@@ -173,14 +173,18 @@ func (m *EncodedMotor) makeAdjustments(ctx context.Context, goalRPM, goalPos, di
 		var currentRPM float64
 		if deltaTime == 0.0 {
 			currentRPM = 0
+			zeroRPMTracker++
+			m.logger.Debug("zero time delta calculated between motor power adjustments")
 		} else {
 			currentRPM = deltaPos / deltaTime
 			if currentRPM == 0 {
 				zeroRPMTracker++
 			}
 		}
-		if zeroRPMTracker > 10 {
-			m.logger.Errorf("error running motor %v, desired rpm is too low for stable motion: %v", m.Name().Name, goalRPM)
+		if zeroRPMTracker > 20 {
+			m.logger.Warnf("motor running at too low an rpm %v for stable motion: %v, trying to run at %v rpm, check if stalled",
```

Comment:
I like this! do you think we should add something about increasing the requested speed? I know for this ticket specifically, it was happening on all requested speeds, but I think in general this warning will appear when a user requests a speed that is too slow. maybe just something like "check if stalled or try increasing rpm". or if you think it should stay more generic like this, I'm fine with that too

Reply:
I like that too! Adding
```suggestion
			m.logger.Warnf("motor running at too low an rpm %v for stable motion: %v, trying to run at %v rpm, check if stalled or try increasing the motor's rpm",
```